### DISCUSSION TITLE
mol_to_smiles parameters

### DISCUSSION THREAD

**Discussion by Fizzizist** [rdkit/rdkit#4506] [Q&A] [answered]
On the postgres [RDKit Cartridge documentation](https://www.rdkit.org/docs/Cartridge.html) it says that the `mol_to_smiles` function returns the canonical smiles of the mol object. When I use the python `rdkit.Chem.MolToSmiles()` function, the `isomericSmiles` parameter defaults to `True`. Can I assume then that the `mol_to_smiles` function also outputs the isomeric form of the smiles?

**Comment by greglandrum** [comment, marked as answer]
Hi @Fizzizist, yes, `mol_to_smiles()` outputs the canonical SMILES; that includes the `isomericSmiles` information.

**Reply by Fizzizist** [reply]
Good to know, thanks!
